### PR TITLE
Fixed the create-pr test and introduced out folder to install

### PR DESCRIPTION
**Pull request by Skarlso**
### Description

Closes https://github.com/weaveworks/pctl/issues/62

### Checklist
- [x] Added tests that cover your change
- [x] Added/modified documentation as required (such as the `README.md`)
- [x] Made sure the title of the PR is a good description that can go into the release notes
- [x] Added a `kind` label to the PR (e.g. `kind/feature`)

### CODE REVIEW

File: pkg/git/git.go [hunk 1]
```diff
@@ -151,10 +149,10 @@ func (g *CLIGit) Push() error {
 func (g *CLIGit) Add() error {
 	fmt.Println("adding unstaged changes")
 	args := []string{
-		"--git-dir", filepath.Join(g.Location, ".git"),
-		"--work-tree", g.Location,
+		"--git-dir", filepath.Join(g.Directory, ".git"),
+		"--work-tree", g.Directory,
 		"add",
-		g.Filename,
+		".",
```
Comment (Skarlso) [inline pkg/git/git.go:155]: I don't necessarily like this, but we don't know what files it generated so we kind of need to add anything that is pending after install.

File: cmd/pctl/install.go [hunk 2]
```diff
@@ -59,6 +58,12 @@ func installCmd() *cli.Command {
 				DefaultText: "main",
 				Usage:       "The base branch to open a PR against.",
 			},
+			&cli.StringFlag{
+				Name:        "out",
+				DefaultText: "current",
+				Value:       ".",
+				Usage:       "Optional location to output generated files into.",
+			},
```
Comment (aclevername) [inline cmd/pctl/install.go:66]: its slightly misleading, its "the location to create the profile installation directory in", as opposed to the place where the contents of the profile installation go in

Comment (Skarlso) [reply cmd/pctl/install.go:66]: ah, true, that was before I realised it creates a folder. :D

**Review by aclevername — APPROVED** [review]
LGTM!

**Pull request merged**